WTWSMS/events/WtWSMS_zusk_misc_events.txt

```diff
@@ -537,7 +550,7 @@ character_event = {
 			trait = sympathy_christendom
 		}
         modifier = {
-			factor = 5
+			factor = 1.05
```

Comment:
But that isn't what you changed here, you simply altered the MTTH to be less likely for those.